knowledge_storm/rm.py

```diff
@@ -396,11 +426,130 @@ def forward(self, query_or_queries: Union[str, List[str]], exclude_urls: List[st
             related_docs = self.qdrant.similarity_search_with_score(query, k=self.k)
             for i in range(len(related_docs)):
                 doc = related_docs[i][0]
-                collected_results.append({
-                    'description': doc.metadata['description'],
-                    'snippets': [doc.page_content],
-                    'title': doc.metadata['title'],
-                    'url': doc.metadata['url'],
-                })
+                collected_results.append(
+                    {
+                        "description": doc.metadata["description"],
+                        "snippets": [doc.page_content],
+                        "title": doc.metadata["title"],
+                        "url": doc.metadata["url"],
+                    }
+                )
+
+        return collected_results
+
+
+class SerperRM(dspy.Retrieve):
+    """Retrieve information from custom queries using Serper.dev.
+
```

Comment:
Remove Line 443-450 since your implementation of `SerperRM` has already enforced this.